parquet-tools/src/main/java/org/apache/parquet/tools/command/CatCommand.java

```diff
@@ -47,7 +47,13 @@
     Option help = OptionBuilder.withLongOpt("json")
                                .withDescription("Show records in JSON format.")
                                .create('j');
+
+    Option showEmpty = OptionBuilder.withLongOpt("show-empty")
+                        .withDescription("Show fields with null values.")
+                        .create('a');
```

Comment:
The long name of the option and single-letter abbreviation should be related, but "show-empty" does not suggest 'a' at all. Some alternatives you could consider:

- leave "show-empty" but change the short form to 'e' as in _empty_.
- leave 'a' as the short form and rename the long option to something with an 'a', like 'all-values-including-empty' (although it's a bit too long).
- use 'n' for the short form and "show-null-values" or "show-nulls" for the long name.